nym-vpn-x/src-tauri/src/grpc/client.rs

```diff
@@ -338,6 +340,78 @@ impl GrpcClient {
         Ok(response.into_inner())
     }
 
+    /// Get the list of available countries for entry gateways
+    #[instrument(skip_all)]
+    pub async fn entry_countries(&self) -> Result<Vec<Country>, VpndError> {
+        debug!("entry_countries");
+        let mut vpnd = self.vpnd().await?;
+
+        let request = Request::new(ListEntryGatewaysRequest {});
+        let response = vpnd.list_entry_gateways(request).await.map_err(|e| {
+            error!("grpc entry_gateways: {}", e);
+            VpndError::GrpcError(e)
+        })?;
+        debug!("gateways count: {}", response.get_ref().gateways.len());
+
+        let countries = response
+            .get_ref()
+            .gateways
+            .iter()
+            .filter_map(|gateway| {
+                // TODO comment this for now has daemon is not returning any probe data
+                // let check = gateway
+                //     .last_probe
+                //     .as_ref()
+                //     .and_then(|probe| probe.outcome.as_ref())
+                //     .and_then(|probe| probe.as_entry.as_ref())
+                //     .map(|v| {
+                //         if v.can_connect && v.can_route {
```

Comment:
Probably better if we don't stash the commented out code in our database, unless it's a code sample.